clients/drcachesim/tools/view.cpp

```diff
@@ -50,6 +50,7 @@
 
 #include "analysis_tool.h"
 #include "dr_api.h"
+#include "dr_ir_encode.h"
```

Comment:
Mixed feelings about clang-tidy's checks for headers which seem to go too far and do not allow how the DR headers were designed with dr_api.h including the others (this is also how some system headers work, along with other API headers most notably windows.h).  IMHO this include should not be necessary as it goes against the recommended usage in the DR docs.  We may be able to find a way to configure clang-tidy.

Reply:
Yes, clang-tidy is a bit strict. Removed.